index.js

```diff
@@ -776,4 +776,26 @@ is.symbol = function (value) {
   return typeof Symbol === 'function' && toStr.call(value) === '[object Symbol]' && typeof symbolValueOf.call(value) === 'symbol';
 };
 
+/**
+ * is.oneOf
+ * Test if `value` is one of the provided `types`
+ *
+ * @param {Mixed} value value to test
+ * @param {Mixed|String[]} types types to test against
+ * @returns {boolean}
+ */
+is.oneOf = function (value, types) {
+  if (!is.array(types) || is.array.empty(types)) {
+    throw new TypeError('second argument must be an array with at least one type');
+  }
+
+  for (var i = 0; i < types.length; i++) {
+    var fn = is[types[i]];
```

Comment:
something was bothering me about this PR, and i think this is the line.

What if someone passes the type "oneOf"? What if they pass the type `toString` or `valueOf` or `hasOwnProperty` or `propertyIsEnumerable`?

I don't think I'm comfortable with dynamically pretending "types" are the same thing as "properties on `is`". What other ideas can we come up with for this?